### PR TITLE
.Net Gemini/vision sample

### PR DESCRIPTION
**Pull request by Krzysztof318**
### Description

vision sample
state: #4680

@RogerBarreto 

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows the [SK Contribution Guidelines](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md) and the [pre-submission formatting script](https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md#development-scripts) raises no violations
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

File: dotnet/samples/KernelSyntaxExamples/Example87_GeminiVision.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+        var chatHistory = new ChatHistory();
+        var chatCompletionService = kernel.GetRequiredService<IChatCompletionService>();
+
+        chatHistory.AddUserMessage(new ChatMessageContentItemCollection
+        {
+            new TextContent("What’s in this image?"),
+            // Vertex AI Gemini API supports both base64 and URI format
+            // You have to always provide the mimeType for the image
+            new ImageContent(new BinaryData(EmbeddedResource.ReadStream("sample_image.jpg")),
```
Comment (SergeyMenshykh) [inline dotnet/samples/KernelSyntaxExamples/Example87_GeminiVision.cs:92]: If the mimeType is always required for an image, should we model it through a dedicated 'string MimeType {get; set;}' property on the ImageContent type and allow to set it via a parameter of class constructors?

Comment (Krzysztof318) [reply dotnet/samples/KernelSyntaxExamples/Example87_GeminiVision.cs:92]: @SergeyMenshykh I think this topic is more complicated. I raised the topic of ImageContent changes at https://github.com/microsoft/semantic-kernel/issues/4759 but did not get a concrete answer.

Comment (RogerBarreto) [reply dotnet/samples/KernelSyntaxExamples/Example87_GeminiVision.cs:92]: BinaryData has the `MediaType`property, do we really need to have an extra metadata for this?

```diff
-new ImageContent(new BinaryData(EmbeddedResource.ReadStream("sample_image.jpg")),
+new ImageContent(new BinaryData(EmbeddedResource.ReadStream("sample_image.jpg"), "image/jpeg")),
```

Comment (Krzysztof318) [reply dotnet/samples/KernelSyntaxExamples/Example87_GeminiVision.cs]: my mistake, fixed

**Review by RogerBarreto — APPROVED** [review]

**Pull request merged**